@xen-orchestra/web-core/lib/components/LegendTitle.vue

```diff
@@ -0,0 +1,39 @@
+<!-- v1.0 -->
+<template>
+  <div class="legend-title typo c3-semi-bold">
+    {{ title }}
+    <UiIcon
+      v-tooltip="{
+        placement: 'top',
+        content: tooltip ? $t(tooltip) : '',
+      }"
+      :icon="tooltip ? icon : undefined"
+      color="info"
+    />
```

Comment:
- The display of the icon should not be conditioned by the presence of the tooltip.
- Tooltip is displayed on top by default so you can directly pass the value.
- Translation of a string prop should not be done by the component.
- Use meaningful class name.

```suggestion
    <UiIcon v-tooltip="iconTooltip ?? false" :icon color="info" class="tooltip-icon" />
```